catroid/src/main/res/layout/dialog_stage.xml

```diff
@@ -115,4 +115,33 @@
         android:contentDescription="@string/stage_dialog_maximize"
         android:src="@drawable/stage_dialog_button_stretch_selector" />
 
+    <LinearLayout xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:theme="@style/Theme.AppCompat"
```

Comment:
why do you have to set the theme here explicitly? this is a little sketchy, because theming and styling in Android is something you don't want to mess with, trust me, you can only loose. One of the reasons is that it behaves differently on every API level and that it is a bit like CSS where some properties are inherited and some are passed down, all while you can never be sure which settings have precedence over others.

Reply:
the constructor of the floating action button checks for the AppCompat theme, otherwise it throws an error which says "You need to use a Theme.AppCompat theme (or descendant) with the design library."